File: PHP/program/Human.php
<?php

class Human
{
    protected $nik = "";
    protected $nama = "";
    protected $jenkel = "";

    public function __construct($nik = "", $nama = "", $jenkel = "")
    {
        $this->nik = $nik;
        $this->nama = $nama;
        $this->jenkel = $jenkel;
    }

    public function getNIK()
    {
        return $this->nik;
    }

    public function setNIK($nik)
    {
        $this->nik = $nik;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function getJenkel()
    {
        return $this->jenkel;
    }

    public function setJenkel($jenkel)
    {
        $this->jenkel = $jenkel;
    }
}

?>

File: CPP/program/Human.cpp
#include <bits/stdc++.h>

using namespace std;

class Human
{
    protected:

        string nik;
        string nama;
        string jenkel;

    public:

        Human(){}

        ~Human(){}

        string getNIK()
        {
            return nik;
        }

        void setNIK(string nik)
        {
            this->nik = nik;
        }

        string getNama()
        {
            return nama;
        }

        void setNama(string nama)
        {
            this->nama = nama;
        }

        string getJenkel()
        {
            return jenkel;
        }

        void setJenkel(string jenkel)
        {
            this->jenkel = jenkel;
        }
};